.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: ubuntu:latest
+    steps:
+      - checkout
+      - run:
+          name: "Set geographic location"
```

Comment:
Why is this necessary?

Reply:
Checkout checks out the branch and downloads it to the docker instance. Since the docker instance is a fresh environment it has no geographic data set, so when trying to install cmake it will ask you to set a location which you cannot to since you cant interact with the terminal in the pipeline.

Reply:
> when trying to install cmake it will ask you to set a location
good to know, thanks!